manifests/d/Daytonaio/Daytona/8.84.0/Daytonaio.Daytona.installer.yaml

```diff
@@ -0,0 +1,14 @@
+# Created using wingetcreate 1.6.1.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.6.0.schema.json
+
+PackageIdentifier: Daytonaio.Daytona
+PackageVersion: 8.84.0
+InstallerType: portable
+Commands:
+- daytona
+Installers:
+- Architecture: x64

```

Comment:
This is the wrong binary, this is the commercial product. The binary that we should package is found here: https://download.daytona.io/daytona/ 

The current version is:
https://download.daytona.io/daytona/v0.3.1/daytona-windows-amd64.exe

We should add the arm build as well.
https://download.daytona.io/daytona/v0.3.1/daytona-windows-arm64.exe

Reply:
Implemented requested changes in this [PR.](https://github.com/microsoft/winget-pkgs/pull/144264)